add-ons/ansible-service-broker/README.md

```diff
@@ -1,71 +1,84 @@
-# Ansible Service Broker Add-on
-
+Ansible Service Broker Addon
+======================
 
 This addon installs the [Ansible Service Broker](https://github.com/openshift/ansible-service-broker)
 
-## Prerequisites
+# Prerequisites
+
+This addon is designed for an RBAC enabled Openshift cluster of `>=3.7`. Minishift can be configured to
```

Comment:
`This addon is designed for a RBAC enabled OpenShift cluster of ...`

Reply:
It feels silly defending the original grammar, but here I go. In this scenario RBAC is an initialism, making the original correct.

Reply:
I'll just pretend like I knew what I was doing :)